yb-voyager/cmd/exportSchema.go

```diff
@@ -57,9 +63,13 @@ func exportSchema() {
 			if !proceed {
 				return
 			}
+			//clear directories
 			for _, dirName := range []string{"schema", "reports", "temp", "metainfo/schema"} {
 				utils.CleanDir(filepath.Join(exportDir, dirName))
 			}
+			//clear files
+			utils.ClearMatchingFiles(filepath.Join(exportDir, META_INFO_DIR_NAME, "metaInfo.json"))
```

Comment:
No need to clear the miginfo.json. Its relevant contents will be anyway overwritten once `export schema` completes.
Assume that in future other operations (e.g. export data) can also put some values in the `miginfo.json`.